source/user-manual/registering/index.rst

```diff
@@ -1,63 +1,127 @@
-.. Copyright (C) 2020 Wazuh, Inc.
+.. Copyright (C) 2019 Wazuh, Inc.
 
 .. _register_agents:
 
-Registering agents
-==================
+Registering Wazuh agents
+========================
 
 .. meta::
-  :description: Learn more about the different methods that can be used to register agents against the Wazuh manager.
+  :description: Learn more about the different methods that can be used to register Wazuh agents against the Wazuh manager.
 
-In this section, the registration process is described, and more specifically the different methods that users can use to register agents in the Wazuh manager.
+By default, Wazuh agents communicate with the Wazuh manager using outbound TCP/UDP connections to two different services:
```

Comment:
I think that we need to improve this part. 
`two different services` and then `1515/TCP: service used for Wazuh agents`. This is not a service. `1515/TCP` is the used port `1515` with protocol `TCP` that `ossec-auth` use by default for listening register requests.